src/index.js

```diff
@@ -91,15 +93,73 @@ function shouldLoadFile(filename) {
   return filename.match(/[-_]test\.js/gi);
 }
 
-function hasMetadataDeclaration({ node }) {
-  if (node.expression && node.expression.type === 'AssignmentExpression') {
-    return (
-      node.expression.left.object.name === 'testMetadata' &&
-      node.expression.left.property.name === 'filePath'
-    );
-  } else {
+function hasMetadataDeclaration({ node }, t) {
+  if (
+    !node.expression ||
+    !t.isAssignmentExpression(node.expression) ||
+    !node.expression.left.object ||
+    !node.expression.left.property
+  ) {
     return false;
   }
+
+  return (
+    node.expression.left.object.name === 'testMetadata' &&
+    node.expression.left.property.name === 'filePath'
+  );
+}
+
+function getNodeProperty(node, path) {
```

Comment:
I'd probably have this in a separate util with some quick tests to sanity check.